poe-tasks/build-and-publish-java-connectors-with-tag.sh

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
+# Usage examples:
+#   ./get-modified-connectors.sh --prev-commit --json | ./build-and-publish-java-connectors-with-tag.sh
+#
+# Specific to this script:
+#   1) Default (main-release) on a single connector
+#   ./build-and-publish-java-connectors-with-tag.sh foo-conn
+#   ./build-and-publish-java-connectors-with-tag.sh --name=foo-conn
+#
+#   2) Explicit main-release with multiple connectors
+#   ./build-and-publish-java-connectors-with-tag.sh --publish-option=main-release foo-conn bar-conn
+#
+#   3) Pre-release (dev tag) via JSON pipe
+#   echo '{"connector":["foo-conn","bar-conn"]}' | ./build-and-publish-java-connectors-with-tag.sh --publish-option=pre-release
+#
+#   4) Mixed: positional + pre-release
+#   ./build-and-publish-java-connectors-with-tag.sh --publish-option=pre-release foo-conn
+set -euo pipefail
+
+CONNECTORS_DIR="airbyte-integrations/connectors"
+
+# ------ Defaults & arg parsing -------
+publish_mode="main-release"
```

Comment:
this feels a little scary? i.e. if I typo `---pre-release`, I think this would actually do a main-release?

Reply:
interesting, I didn't consider this. I'll reverse it.

Reply:
Reversed.